.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,70 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+  pull_request:
+
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Code style checks
+            os: ubuntu-latest
+            python: 3.x
+            toxenv: codestyle
+
+          - name: Build the docs
+            os: ubuntu-latest
+            python: 3.8
+            toxenv: build_docs
+
+          - name: Python 3.9 with minimal dependencies
+            os: ubuntu-latest
+            python: 3.9
+            toxenv: py39
+
+          - name: Python 3.9 with full coverage
+            os: ubuntu-latest
+            python: 3.9
+            toxenv: py39-alldeps-cov
+
+          - name: Python 3.6 with oldest supported version of all dependencies
+            os: ubuntu-latest
+            python: 3.6
+            toxenv: py36-oldestdeps
+
+          - name: Python 3.7 (MacOS X) with all optional dependencies
+            os: macos-latest
+            python: 3.7
+            toxenv: py37-alldeps
+
+          - name: Python 3.8 (Windows) with all optional dependencies
```

Comment:
Why is it python 3.8 on Windows but 3.7 on MacOS?

Reply:
I did that so that we would be running at least one test on each supported Python version (3.6, 3.7, 3.8, and 3.9). Astropy does the same thing.

Reply:
Seems good. Is there something we can do to try and notify us that we need to add a new test once a new python version comes out?